### PR TITLE
[065] Update to webpack 2

### PR DESCRIPTION
**Pull request by mikeerickson**
- [x] Convert `webpack.config.js` to work with webpack 2
- [x] Update all associated dependencies (see diff for `package.json`)
- [x] Updated `README.md` to reference `webpack 2` instead of `webpack`
- [x] Updated version from `1.0.0 -> 1.1.0` (maintaining semver)
- [x] Added `postcss.config.js` (see file comments for reasoning)
NOTE: I have not updated karma testing as there are some remaining issues to cure

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -139,18 +148,24 @@ module.exports = function makeWebpackConfig () {
    * Reference: https://github.com/postcss/autoprefixer-core
    * Add vendor prefixes to your css
    */
-  config.postcss = [
-    autoprefixer({
-      browsers: ['last 2 version']
-    })
-  ];
+   // NOTE: This is now handled in the `postcss.config.js`
+   //       webpack2 has some issues, making the config file necessary
```
Comment (Foxandxss) [inline webpack.config.js]: explain please

Comment (mikeerickson) [inline webpack.config.js]: I will find the reasoning behind this and send it to you.   I applied this fix personally sometime ago (when working with webpack beta release) and have not revistited

Comment (mikeerickson) [inline webpack.config.js]: First and formost, `webpack 2` does not allow "custom" config properties (can only use webpack native properties).   

So, the `postcss.config.js` replaces the use of 

```
...
   config.postcss()
...
```

Comment (Foxandxss) [inline webpack.config.js]: What I ask is why an external file, I prefer everything contained.

Comment (mikeerickson) [inline webpack.config.js]: Like I mentioned above, I will track down where the information I gleaned as to requiring the `postcss.config.js`   I understand what you are asking, but I am just stating that I dont recall where I say this (when I had to add it to my project).

Further, adding the `postcss.config.js` is the usage of `.babelrc` (it too could be stored in the `package.json`

I will track down where I found the information about needing to use `postcss.config.js` with webpack 2

Comment (mikeerickson) [inline webpack.config.js]: @Foxandxss I believe this is where i got my information (near bottom of thread, stating `postcss.config.js` is required when using postcss plugins)

https://github.com/postcss/postcss-loader/issues/125

image

Comment (Foxandxss) [inline webpack.config.js]: I am good with that. Can you resolve the conflict tho?

Comment (mikeerickson) [inline webpack.config.js]: Sure thing...

**Comment by mikeerickson** [comment]
@Foxandxss OK, merge conflicts fixed

**Comment by Foxandxss** [comment]
Thanks.

**Comment by phra** [comment]
nice one! @mikeerickson

**Pull request merged**